daisy_integration_tests/daisy_e2e.test.gotmpl

```diff
@@ -35,9 +35,6 @@
     "Windows Server 2019 translate": {
       "Path": "./2019_translate.wf.json"
     },
-    "Sources upload/download": {
```

Comment:
idk the context of this file, can you give a brief explanation why it isn't being deleted like the rest?

Reply:
What remains are import/export tests. As the WAW team moves to their own repo, I left these for them to decide whether to keep them, and how they'd like to name them. (The name "daisy e2e" is somewhat fitting in the context of import/export since these tests are running the daisy workflows that are used by import/export.)